library/src/commonMain/kotlin/edu/usf/cutr/otp/plan/api/PlanApi.kt

```diff
@@ -45,11 +45,16 @@ class PlanApi(private val url: String,
 
         GlobalScope.launch(ApplicationDispatcher) {
             try {
-                val parameters = buildParameters(requestParameters)
-                val url = URLBuilder(
-                    parameters = parameters,
-                ).takeFrom(url).buildString()
-                val json = HttpClient().get<String>(url)
+                val urlString = if (requestParameters != null) {
```

Comment:
Same on keeping `requestParameters` non-nullable - we should be able to revert this